JavaScript/Github_unfollowers/Github_Unfollowers.js

```diff
@@ -0,0 +1,38 @@
+// Script to track recent unfollowers on github using github api
```

Comment:
Replace `Script to track recent unfollowers on github using github api` to `Script to track recent people who unfollowed you on Github using the Github API`